channels/drdynvc/client/drdynvc_main.c

```diff
@@ -409,11 +409,25 @@ static UINT dvcman_write_channel(IWTSVirtualChannel* pChannel, ULONG cbSize,
 static UINT dvcman_close_channel_iface(IWTSVirtualChannel* pChannel)
 {
 	DVCMAN_CHANNEL* channel = (DVCMAN_CHANNEL*) pChannel;
+	DrdynvcClientContext* context;
+	UINT error = CHANNEL_RC_OK;
+	WLog_INFO(TAG, "close_channel_iface: id=%"PRIu32"", channel->channel_id);
 
-	if (!channel)
-		return CHANNEL_RC_BAD_CHANNEL;
+	if (channel->status == CHANNEL_RC_OK)
```

Comment:
Refer: urbdrc_search_usb_device
When usb device was removed, channel->Close() was closed to close the channel for this device.
As the channel->Close reference to dvcman_close_channel_iface.
According to the RDPEUSB document, we should remove the channel, So I copied some lines from dvcman_close_channel to dvcman_close_channel_iface;

I missed the NULL check - -||